### PR TITLE
Update to use node LTS release.

### PR DESCRIPTION
**Pull request by aellison**

### CODE REVIEW

**Comment by skippednote** [comment]
@aellison Instead of hard coding the version number should we use something like this?
```
nvm install lts/*
nvm alias default lts/*
```

This will keep the repository upto date without breaking anything as the updates to node version would be **minor** or **patch** level. LTS version will change after a year, so we won't need to update anything (docs mostly) for that time period.

**Comment by briantully** [comment]
Agreed @skippednote !  

Even with the proposed updates to the docs, we're specifying "6.9.0" which is out of date to the current 6.11.0.

installing 'lts/boron' would be an alternative approach.

`nvm install lts/boron`
`nvm use lts/boron`
`nvm alias default lts/boron`

**Comment by jenter** [comment]
@aellison have you had a chance to look at the Travis fail

**Comment by sarahjean** [comment]
I think I like this `lts/boron` suggestion, wasn't aware that you could do that!

**Comment by briantully** [comment]
it looks like the issue is that yarn is not installed.

do we need to explicitly install it via Cog rather than BLT?
i.e., 

`npm install --global yarn`

**Comment by sarahjean** [comment]
@briantully we just merged the PR to go back to NPM and use latest, so that shouldn't be a problem in a sec. https://github.com/acquia-pso/cog/pull/100

File: STARTERKIT/README.md [hunk 1]
```diff
@@ -38,8 +38,7 @@ Once you have created a custom sub-theme, you will setup for local compiling. If
 * Install Node.js with `./install-node.sh 4.4.4` and then point to the proper version with `source ~/.bashrc && nvm use --delete-prefix 4.4.4`
```
Comment (sarahjean) [inline STARTERKIT/README.md]: This should reference the newer version of node as well.

**Comment by lkopacz** [comment]
@aellison it seems that this is still failing due to the syntax of `nvm use lts/boron`

**Pull request closed without merging**